tracer/build/_build/Build.GitHub.cs

```diff
@@ -268,8 +269,7 @@ partial class Build
                 string line;
                 while ((line = reader.ReadLine()) != null)
                 {
-                    if (!line.StartsWith("⚠ 1. Download the NuGet packages")
-                     && !line.StartsWith("⚠ 2. Download the signed"))
+                    if (!line.StartsWith("⚠ 1. Download the NuGet packages"))
```

Comment:
I think you can remove all of this now, essentially reverting https://github.com/DataDog/dd-trace-dotnet/pull/2324/commits/264acd443fc8852e8f0da72cfd1b361a6c397943?

Reply:
Done in 1ad8ac42a336fecd7aca60a84fe6b98aca0109d7